### PR TITLE
Implement support for multiple monitors

### PR DESCRIPTION
**Pull request by milkdropper**
This PR implements support for multiple monitors (ALT+SHIFT).

It also updates the help menu, where a toast message incorrectly labeled it as F1 when it was really F12. I also removed the portions of the help menu that don't work as a standalone application.

### CODE REVIEW

**Comment by OfficialIncubo** [comment]
Please try my improvement one: https://github.com/OfficialIncubo/BeatDrop-Music-Visualizer

I have changed the ALT+SHIFT to ALT+S starting at version 1.3 because ALT+SHIFT is a shortcut for changing keyboard language.
Download, Install or Extract, Open BeatDrop, press ALT+S and you're good to go!
Also, if you want to try more features, press "F1" for help.

**Review by Chriswsi — CHANGES_REQUESTED** [review]
> [](url)

**Comment by OfficialIncubo** [comment]
Huh?

TL;DR Try: https://github.com/OfficialIncubo/BeatDrop-Music-Visualizer
This feature has already implemented in. You can try ALT + S or you can start the program with this feature based of .ini setting file (beatdrop.ini)
